### DISCUSSION TITLE
Reverse Proxy on Apache

### DISCUSSION THREAD

**Discussion by resoftw** [filamentphp/filament#7054] [Help]
My filament is on other server, and I'd like to publish it on my main domain under a subfolder: mydomain.com/myfilament, because that's the domain that I can access from outside (public ip)..
How to setup filament to run behind reverse proxy?, I'm using apache 2.4
thank's

**Comment by leandrocfe** [comment]
Have you ever done this using another laravel project? If it works with Laravel, it is supposed to work with Filament.

**Reply by resoftw** [reply]
yes, I've tried with inertia, livewire, they're work...
